### PR TITLE
Add about us fields

### PR DESCRIPTION
**Pull request by RubenSousaDinis**
## Summary

This PR adds new fields to the about section of the talent profile. This will add more relevant data so we can analize our user base better.

## Notion link

https://www.notion.so/talentprotocol/Profile-Add-basic-about-fields-b90fa5cfbd6349db8ee41c1c138c857f

### CODE REVIEW

**Review by verdefred — COMMENTED** [review]
great work just left a couple of comments

File: app/packs/src/components/talent/Edit/About.jsx [hunk 1]
```diff
@@ -206,7 +226,7 @@ const About = (props) => {
   };
 
   const cannotSaveProfile =
-    !props.talent.profile.headline || !props.talent.profile.occupation || !props.profilePictureUrl;
+    !props.talent.profile.headline || !props.talent.profile.occupation;
```
[the remark this answers is not in the document]
Comment (RubenSousaDinis) [reply app/packs/src/components/talent/Edit/About.jsx:229]: good spot. removed for testing

File: db/migrate/20220609160644_add_open_to_offers_to_talent.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddOpenToOffersToTalent < ActiveRecord::Migration[6.1]
+  def change
+    add_column :talent, :open_to_job_offers, :boolean, default: false
```
Comment (verdefred) [inline db/migrate/20220609160644_add_open_to_offers_to_talent.rb:3]: can we also add `null: false`?

**Review by verdefred — APPROVED** [review]
🚀

**Pull request merged**